src/settings-ui/Settings.UI/Views/ShellPage.xaml

```diff
@@ -10,8 +10,9 @@
     xmlns:ic="using:Microsoft.Xaml.Interactions.Core"
     xmlns:i="using:Microsoft.Xaml.Interactivity"
     HighContrastAdjustment="None"
-    muxc:BackdropMaterial.ApplyToRootOrPageBackground="True"
     mc:Ignorable="d">
+    <!--TODO(stefan): Bring back Should be available in v1.1
+    BackdropMaterial.ApplyToRootOrPageBackground="True"-->
```

Comment:
Actually BackDrop material seemed to be what was causing a Access denied exception on some versions of Windows 10 with XAML islands. It doesn't  occur anymore with this WinUI3 change, but it might have been because of this removal as well.

Reply:
It's for Mica material, which we don't use/show anyway.. So it's a safe removal